t/t-multiple-remotes.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+. "$(dirname "$0")/testlib.sh"
+
+reponame="$(basename "$0" ".sh")"
+
+prepare_mrtest(){
+  MRHOME=$TRASHDIR/home/mrhome
+  rm -rf $MRHOME
+  mkdir $MRHOME
+  HOME=$MRHOME
+  git config --global user.name "Git LFS Tests"
+  git config --global user.email "git-lfs@example.com"
+}
```

Comment:
I don't think we should be setting `HOME` here, since the test library does that, and we also don't need to set the Git config values, since those should already be set.

Also, please here and everywhere else, double-quote all variables.  On Windows, it's extremely common for a user's home directory (and thus all paths to their repositories) to have spaces, and we need to handle that gracefully.

Reply:
You are right. I added a comment to explain my motivation for that. 
`The global configuration sets a common http endpoint (ip:port) for lfs`
 `We deactivate the git global configuration related with the http protocol`
 `by setting a new .gitconfig file`
 Additionally, I am not setting a new $HOME. Instead I am removing .gitconfig file and setting the settings I need. At the begging of the next test the original .gitconfig file is restored by the function begin_test()

Reply:
I'm sorry, I still think there's something I'm missing.  What specific setting is causing the problem?  In `t/t-standalone-file.sh`, we use `file:///` URLs just fine, so it can't be `lfs.url`.

Can we keep the regular `.gitconfig` and simply unset the offending option in each test with `git config --global --unset`?  I'd rather we do that because it means that if we add an important new setting into the testsuite in the future, the entire testsuite picks it up.

Reply:
I am sorry. I seems that it was not necessary to chage the global configuration after all. Thank you for pointing that out. The new patch uses the global seetings provided by `setup()`